versioned/storage/cache/src/test/java/org/projectnessie/versioned/storage/cache/TestCacheOvershoot.java

```diff
@@ -123,9 +123,14 @@ private void testCacheOvershoot(Executor evictionExecutor) throws Exception {
       executor.awaitTermination(10, TimeUnit.MINUTES);
     }
 
+    // The read of `Eviction` properties is "just" a volatile read since Caffeine 3.2.3
+    // and trigger cleanups asynchronously. Before 3.2.3, cleanups happened synchronously.
+    // This change breaks the initially present assertions of this test, but not the
```

Comment:
"This change" is good in a commit message, but we want to keep it in the comment (forever)?